CareKit/CareCard/OCKCareCardViewController.h

```diff
@@ -144,6 +145,11 @@ OCK_CLASS_AVAILABLE
  */
 @property (nonatomic, readonly, nullable) OCKCarePlanEvent *lastSelectedInterventionEvent;
 
+/** 
+ A reference to the `UITableView` contained in the view controller
+ */
+@property (nonatomic, nonnull) UITableView *tableView;
```

Comment:
When you make the tableView readonly, it does not apply to any of its properties. The properties have their own read/write specifier. As long as the tableViewHeader is not defined as readonly (which it is not), setting the tableView as readonly should still allow you to set the tableViewHeader.